DOCS/interface-changes.rst

```diff
@@ -31,6 +31,7 @@ Interface changes
 ::
 
  --- mpv 0.41.0 ---
+    - add `--input-tablet-emulate-mouse` option
```

Comment:
This should be separate file in interface-changes directory. Don't edit rst directly.